src/components/Postiterie.vue

```diff
@@ -207,16 +207,20 @@ export default {
 
     sendFile: function(){
       let $this = this;
```

Comment:
This is a code smell. And it's probably not needed.

Reply:
Indeed this one wasn't needed but I had to use it in the tracking function, is it a code smell because of the name of the var or is there a better way to access the content of a "parent this" ?

Reply:
In general, accessing "parent this" is a code smell because it creates a tight coupling between parents and children. And tight coupling is the enemy of reusability and maintainability.
If you child need a value, give it. If you need to call a method from your parent or update its data, it's probably better to use an event.